docs/source/overview/options.md

```diff
@@ -152,19 +152,23 @@ information of the evaluator contains following items:
 
     - `type: [str]` The type of the metric. The supported types are `accuracy`, `latency` and `custom`.
 
-    - `subtype: [str]` The subtype of the metric. Please refer to [AccuracySubtype](accuracy_sub_type) and
-    [LatencySubtype](latency_sub_type) for the supported sub-types. It is `null` for `custom` type.
+    - `subtypes: [List[Dict]]` The subtypes of the metric. Cannot be null or empty. Each subtype is a dictionary that contains following items:
 
-    - `higher_is_better: [Boolean]` True if the metric is better when it is higher. It is `true` for `accuracy` type and `false` for `latency` type.
+        - `name: str` The name of the subtype. Please refer to [AccuracySubtype](accuracy_sub_type) and [LatencySubtype](latency_sub_type)
+        for the supported sub-types. For `custom` type, if the result of the evaluation is a dictionary, the name of the subtype should be the key of the dictionary. Otherwise, the name of the subtype could be any unique string user gives.
 
-    - `goal: [Dict]` The goal of the metric. It is a dictionary that contains following items:
+        - `priority_rank: [int]` The priority rank of the subtype. The subtype with the highest priority rank will be used to evaluate the. Note that it should be unique among all subtypes in the metric.
```

Comment:
the term priority is similar with rank, could we use one?

Reply:
Updated to priority.